src/Poll.ts

```diff
@@ -83,7 +80,6 @@ export class Poll {
         if (optionArray[0].toLowerCase() === "anon" || optionArray[1].toLowerCase() === "anon") {
             selection.options!.push(this.buildSelectOption("Collect Results", "collect"));
         }
-        actionBlockCount++;
```

Comment:
Why don't we increment this here?

Reply:
Because the variable is never subsequently called